src/main/java/org/casbin/jcasbin/util/Util.java

```diff
@@ -107,6 +107,25 @@ public static String escapeAssertion(String s) {
         return sb.toString();
     }
 
+    /**
+     * convertInSyntax Convert 'in' to 'seq' to fit aviatorscript,because aviatorscript don't support native 'in' syntax
+     *
+     * @param expString the value of the matcher
+     * @return the 'seq' expression.
+     */
+    public static String convertInSyntax(String expString) {
+        String reg = "((r|p)[0-9]*_[a-zA-Z]*.([a-zA-Z]*)) *?in *?\\((.*?)\\)";
+        Matcher m1 = Pattern.compile(reg).matcher(expString);
+        StringBuffer sb = new StringBuffer();
+        boolean flag=false;
+        while (m1.find()) {
+            flag=true;
+            String s = m1.group();
+            s = s.replaceAll(reg,"seq.some($4, fn(x) {x.$3 == $1}) != nil");
+            m1.appendReplacement(sb,s);
```

Comment:
This looks very odd and unnecessary. `s` is the match, so it will for certain match the regex. I think that just calling `m1.appendReplacement(sb, "seq.some($4, fn(x0 {x.$3 == $1}) != nil");` will do the same (but I haven't tested it).

Reply:
you're right ,it can work.now it looks good